project/{{ cookiecutter.__folder_name }}/devops/README.md

```diff
@@ -51,6 +51,12 @@ STACK_NAME={{ cookiecutter.__devops_stack_name }}
 
 Note: The `.env` file is included in `.gitignore`, ensuring environment-specific configurations aren't pushed to the repository.
 
+
+### Server installation
+
+You need to Ubuntu or Debian in {{ cookiecutter.hostname }}, enable SSH and install python3 there.
```

Comment:
This sentence still does not make sense, even after I cleaned up a part of it.

```suggestion
You need to Ubuntu or Debian in {{ cookiecutter.hostname }}, enable SSH, and install a supported version of Python 3 there.
```

Reply:
Perhaps "You need to specify Ubuntu or Debian in..."?

And is "there" in the file `.env`?

I see `ansible_lsb.id` below. Does that figure into this?

Reply:
Yes, the `ansible_lsb.id` figures out if we are using Debian or Ubuntu in the server. No need to set it anywhere.

Reply:
OK. How do we fix this sentence? It still doesn't make sense.